conseil-lorre/src/main/scala/tech/cryptonomic/conseil/indexer/bitcoin/BitcoinIndexer.scala

```diff
@@ -120,8 +93,8 @@ class BitcoinIndexer(
       rpcClient <- RpcClient.resource(
         bitcoinConf.node.url,
         maxConcurrent = bitcoinConf.batching.indexerThreadsCount,
-        httpClient, // TODO: wrap it into retry and logger middleware
-        Authorization(BasicCredentials(bitcoinConf.node.username, bitcoinConf.node.password))
+        httpClient // TODO: wrap it into retry and logger middleware
+        // Authorization(BasicCredentials(bitcoinConf.node.username, bitcoinConf.node.password))
```

Comment:
can you explain why is this commented?

Reply:
types in the new version of API were incompatible; but I managed to fix it now